src/Setup/DbUtil.php

```diff
@@ -29,7 +30,7 @@ public static function encodeDsn($db) {
     return sprintf('mysql://%s:%s@%s/%s',
       $db['username'],
       $db['password'],
-      self::encodeHostPort($db['host'], $db['port']),
+      $db['server'],
```

Comment:
won't this cause people with non standard ports to have issues? or is `$db['server']` defined further up

Reply:
@seamuslee001 yes its defined up where `parseDsn(...)` will return the db creds, among which the `server` attribute will be first encoded with host and port information. See [here](https://github.com/civicrm/civicrm-setup/pull/9/files#diff-80d357fa07f6de080b7f7277a7fc4ad0R16)